taf/auth_repo.py

```diff
@@ -173,6 +174,18 @@ def get_metadata(
         else:
             return self.get_json(commit, metadata_path)
 
+    def get_info_json(
+        self, commit: Optional[str] = None, safely: bool = True
+    ) -> Optional[Dict]:
+        if commit is None:
+            commit = self.head_commit_sha()
```

Comment:
suggestion: You can remove the need for the first if statement by doing `head_commit =  commit or self.head_commit_sha()` and then using `head_commit` variable?

Reply:
Applied suggestion.